core/templates/dev/head/services/ExtensionTagAssemblerService.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
year :)

Reply:
I should change it to 2018?
I went through a merged PR for this issue(#3826) - Found 2014 in it(#4414).